server/models/questions/removeReader.js

```diff
@@ -0,0 +1,36 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {String} param0.reader
+ * @param {Number} param0.questionId
+ * @return {Promise}
+ */
+
+function removeReader({ username, reader, questionId }) {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      `DELETE FROM questions_editors a WHERE
+       EXISTS(SELECT 1 FROM (SELECT * FROM questions_editors) b WHERE editor=? AND question_id=?)
+       AND editor=? AND access=?`,
```

Comment:
#### CHANGE

1. the person **`a`** who is removing reader **`b`** can have read or write access, so no need of checking that.
2. if the reader **`b`** will have `read` access then there is no need of checking if he is the creator or not cuz the creator will have `write` access.

Reply:
@cjchirag7  I think only `editor` should be allowed to remove a `reader`. Thoughts ?

Reply:
That makes sense. @ridhishjain , please update the PR to allow only editors to add or remove a `reader`.